### PR TITLE
Remove unneeded config from the ember-cli-build.js file

### PR DESCRIPTION
**Pull request by Windvis**

### CODE REVIEW

File: ember-cli-build.js [hunk 1]
```diff
@@ -2,48 +2,11 @@
...
-          global: true,
-        },
-      },
-      useSwcParser: true,
-      resolve: {
-        extensions: ['.js', '.json'],
-      },
-    },
-    hinting: IS_TEST,
-    tests: IS_TEST,
```
Comment (Windvis) [inline ember-cli-build.js:24]: This disables the `/tests` route which is very handy.

File: ember-cli-build.js [hunk 2]
```diff
@@ -2,48 +2,11 @@
...
-        node: {
-          global: true,
-        },
-      },
-      useSwcParser: true,
-      resolve: {
-        extensions: ['.js', '.json'],
-      },
-    },
-    hinting: IS_TEST,
```
Comment (Windvis) [inline ember-cli-build.js:23]: Not sure what this does exactly.

File: ember-cli-build.js [hunk 3]
```diff
@@ -2,48 +2,11 @@
...
-    tests: IS_TEST,
-    'ember-cli-babel': {
-      includePolyfill: IS_PROD,
-    },
```
Comment (Windvis) [inline ember-cli-build.js:27]: I don't think the polyfill is needed? Assuming we only support modern browsers?

File: ember-cli-build.js [hunk 4]
```diff
@@ -2,48 +2,11 @@
...
-    autoprefixer: {
-      sourcemap: false,
-    },
```
Comment (Windvis) [inline ember-cli-build.js:30]: Not sure why this was needed. If we don't want the autoprefixing we should just remove the ember-cli-autoprefixer addon.

File: ember-cli-build.js [hunk 5]
```diff
@@ -2,48 +2,11 @@
...
-    sourcemaps: {
-      enabled: IS_PROD,
-    },
```
Comment (Windvis) [inline ember-cli-build.js:33]: It seems strange to disable source maps in development builds so I would either always enable them, or keep the default behaviour (enabled in dev, not in prod).

File: ember-cli-build.js [hunk 6]
```diff
@@ -2,48 +2,11 @@
...
-    fingerprint: {
-      enabled: true,
-    },
```
Comment (Windvis) [inline ember-cli-build.js:36]: We don't need this in development.

File: ember-cli-build.js [hunk 7]
```diff
@@ -2,48 +2,11 @@
...
     '@appuniversum/ember-appuniversum': {
       disableWormholeElement: true,
     },
-    'ember-cli-terser': {
-      enabled: true,
```
Comment (Windvis) [inline ember-cli-build.js:41]: We really don't want this in dev builds, it slows down hot rebuilds a lot.

File: ember-cli-build.js [hunk 8]
```diff
@@ -2,48 +2,11 @@
...
-      exclude: ['assets/vendor.js', 'flandersMunicipalities.ts'],
-    },
-    'ember-cli-htmlbars-inline-precompile': {
```
Comment (Windvis) [inline ember-cli-build.js:44]: This is superceded by ember-cli-htmlbars, we should also remove the dependency.

File: ember-cli-build.js [hunk 9]
```diff
@@ -2,48 +2,11 @@
...
 const EmberApp = require('ember-cli/lib/broccoli/ember-app');
 
-const environment = process.env.EMBER_ENV;
-const IS_PROD = environment === 'production';
-const IS_TEST = environment === 'test';
-
 module.exports = function (defaults) {
   let app = new EmberApp(defaults, {
-    autoImport: {
-      exclude: ['flandersMunicipalities.ts'],
```
Comment (Windvis) [inline ember-cli-build.js:12]: This doesn't seem to be imported anywhere? Outdated config I guess?

File: ember-cli-build.js [hunk 10]
```diff
@@ -2,48 +2,11 @@
...
-      webpack: {
-        node: {
-          global: true,
```
Comment (Windvis) [inline ember-cli-build.js:15]: The build didn't seem to fail with this removed, so I don't think we need it anymore.

File: ember-cli-build.js (hunk 9 above)
Comment (aliokan) [reply ember-cli-build.js:12]: I guess it was use before using municipality from the server

**Review by aliokan — APPROVED** [review]
Approved! :pray:

**Pull request merged**